src/Common/Exception.cpp

```diff
@@ -400,6 +400,18 @@ PreformattedMessage getCurrentExceptionMessageAndPattern(bool with_stacktrace, b
                 << " (version " << VERSION_STRING << VERSION_OFFICIAL << ")";
         }
         catch (...) {}
+
+// #ifdef ABORT_ON_LOGICAL_ERROR
+//         try
+//         {
+//             throw;
+//         }
+//         catch (const std::logic_error &)
```

Comment:
`std::logic_error` indicates a bug just like our `LOGICAL_ERROR`, but it is thrown in the CI sometimes, so it's commented out. I will try to enable it in another PR